uberAgentSupport-macOS/uberAgentSupport.zsh

```diff
@@ -40,27 +40,56 @@ if [ "$(id -u)" != "0" ]; then
 fi
 
 # Create a structured directory tree inside the bundle path
-mkdir -p "${bundlePath}"/{Config/Local,Config/CCFM,Logs/Daemon,Logs/Users,CrashReports}
+mkdir -p "${bundlePath}"/{Config/Local,Config/CCFM,Logs/Daemon,Logs/Users,Logs/Remote,CrashReports}
 
 # Define a log file path inside the bundle
-logPath="${bundlePath}/uASupportBundleLog.txt"
-
-# Redirect all future command outputs(both stdout and stderr) to the specified log file
-exec > >(tee -a $logPath) 2>&1
+scriptLogPath="${bundlePath}/uASupportBundleLog.txt"
+
+# Redirect all future command outputs (both stdout and stderr) to the specified log file
+exec > >(tee -a $scriptLogPath) 2>&1
+
+# Function to collect log files from either default or specified path
+collectLogs() {
+    local remoteLogfilePath="$1"
+
+    if [ -d "${remoteLogfilePath}" ]; then
+        echo "Collecting logs from ${remoteLogfilePath}..."
+        find "${remoteLogfilePath}" -type f -name "*${hostname}*" -exec cp {} "${bundlePath}/Logs/Remote/" \;
```

Comment:
Do we need error handling here, i.e., could the find command fail?

Reply:
find usually does not fail silently, but of course can fail for various reasons. I think we can leave as is for the time being.